app/handlers/objectTracking.py

```diff
@@ -119,7 +120,19 @@ def run(self):
         try:
             subprocess.check_call(fbttf_call)
             subprocess.check_call(fbtgf_call)
-            subprocess.check_call(["classify-objects.py", "--cfg", tracking_path, "-d", db_path])  # Classify road users
+
+            #Classify Road Users with -s, -n
+            total_objs = getObjectCount(db_path)
+            num_objs = 100
```

Comment:
I think your comment here or variable names could be better.  I like "batch_size" since that's a data pipeline way of describing num_objs.